### ISSUE TITLE
Support cmd-brush to deselect points in line plot

### ISSUE THREAD

**Issue by huddlej** [dms-view/dms-view.github.io#79]
- [x] Allow users to hold the "command" key (or the PC equivalent) and brush to deselect points
- [x] Change the mouse cursor when "command" key is held down, to reflect the change in mode
- [x] Toggle the corresponding select/deselect radio button when the user presses or releases the "command" key

current bug: what happens if the user lets go of the metaKey midbrush

**Comment by skhilton** [comment]
Here is an example of d3 radio buttons
http://jsfiddle.net/jhaRoshan/54559btv/

**Comment by skhilton** [comment]
https://stackoverflow.com/questions/2761455/how-do-you-change-an-html-radiobutton-selection-from-javascript

**Comment by skhilton** [comment]
The brush also creates the SVG elements necessary to display the brush selection and to receive input events for interaction. You can add, remove or modify these elements as desired to change the brush appearance; you can also apply stylesheets to modify the brush appearance. The structure of a two-dimensional brush is as follows:
```
<g class="brush" fill="none" pointer-events="all" style="-webkit-tap-highlight-color: rgba(0, 0, 0, 0);">
  <rect class="overlay" pointer-events="all" cursor="crosshair" x="0" y="0" width="960" height="500"></rect>
  <rect class="selection" cursor="move" fill="#777" fill-opacity="0.3" stroke="#fff" shape-rendering="crispEdges" x="112" y="194" width="182" height="83"></rect>
  <rect class="handle handle--n" cursor="ns-resize" x="107" y="189" width="192" height="10"></rect>
  <rect class="handle handle--e" cursor="ew-resize" x="289" y="189" width="10" height="93"></rect>
  <rect class="handle handle--s" cursor="ns-resize" x="107" y="272" width="192" height="10"></rect>
  <rect class="handle handle--w" cursor="ew-resize" x="107" y="189" width="10" height="93"></rect>
  <rect class="handle handle--nw" cursor="nwse-resize" x="107" y="189" width="10" height="10"></rect>
  <rect class="handle handle--ne" cursor="nesw-resize" x="289" y="189" width="10" height="10"></rect>
  <rect class="handle handle--se" cursor="nwse-resize" x="289" y="272" width="10" height="10"></rect>
  <rect class="handle handle--sw" cursor="nesw-resize" x="107" y="272" width="10" height="10"></rect>
</g>
```
https://github.com/d3/d3-brush/blob/master/README.md

**Comment by skhilton** [comment]
https://stackoverflow.com/questions/18551277/using-external-images-for-css-custom-cursors

**Comment by skhilton** [comment]
closed by PR #91
